src/translation/translation_de_DE.ts

```diff
@@ -1646,12 +1646,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../clientsettingsdlg.cpp" line="394"/>
         <source>Audio Alerts</source>
-        <translation>Audiohinweise</translation>
+        <translation type="unfinished">Audiohinweise</translation>
     </message>
     <message>
         <location filename="../clientsettingsdlg.cpp" line="395"/>
-        <source>Enable audio alert when receiving a chat message and when a new client joins the session. A second sound device may be required to hear the alerts.</source>
-        <translation>Aktiviert Audiohinweise beim Empfang einer Chat-Nachricht bzw. wenn ein neuer Client beitritt. Möglicherweise ist ein zweites Audiogerät erforderlich, um die Hinweise zu hören.</translation>
+        <source>Trigger an audio alert when receiving a chat message and when a new client joins the session. A second sound device may be required to hear the alerts.</source>
```

Comment:
```suggestion
        <source>Trigger an audio alert when receiving a chat message and when a new client joins the session. (A second sound device may be required to hear the alerts.)</source>
```

I'm going through a (parenthetical) phase.

Reply:
Don't edit the `.ts` files, edit the source.